diff --git a/src/Ryujinx.Gtk3/UI/MainWindow.glade b/src/Ryujinx.Gtk3/UI/MainWindow.glade
index d1b6872..4da444d 100644
--- a/src/Ryujinx.Gtk3/UI/MainWindow.glade
+++ b/src/Ryujinx.Gtk3/UI/MainWindow.glade
@@ -545,7 +545,7 @@
                     <property name="can-focus">False</property>
                     <child>
                       <object class="GtkMenuItem" id="UpdateMenuItem">
-                        <property name="visible">True</property>
+                        <property name="visible">False</property>
                         <property name="can-focus">False</property>
                         <property name="tooltip-text" translatable="yes">Check for updates to Ryujinx</property>
                         <property name="label" translatable="yes">Check for Updates</property>
@@ -555,7 +555,7 @@
                     </child>
                     <child>
                       <object class="GtkSeparatorMenuItem">
-                        <property name="visible">True</property>
+                        <property name="visible">False</property>
                         <property name="can-focus">False</property>
                       </object>
                     </child>
diff --git a/src/Ryujinx.Gtk3/UI/Windows/SettingsWindow.glade b/src/Ryujinx.Gtk3/UI/Windows/SettingsWindow.glade
index f0dbd6b..9ba7e50 100644
--- a/src/Ryujinx.Gtk3/UI/Windows/SettingsWindow.glade
+++ b/src/Ryujinx.Gtk3/UI/Windows/SettingsWindow.glade
@@ -130,7 +130,7 @@
                                 <child>
                                   <object class="GtkCheckButton" id="_checkUpdatesToggle">
                                     <property name="label" translatable="yes">Check for Updates on Launch</property>
-                                    <property name="visible">True</property>
+                                    <property name="visible">False</property>
                                     <property name="can-focus">True</property>
                                     <property name="receives-default">False</property>
                                     <property name="halign">start</property>
diff --git a/src/Ryujinx.UI.Common/Configuration/ConfigurationState.cs b/src/Ryujinx.UI.Common/Configuration/ConfigurationState.cs
index b7f3608..3d04928 100644
--- a/src/Ryujinx.UI.Common/Configuration/ConfigurationState.cs
+++ b/src/Ryujinx.UI.Common/Configuration/ConfigurationState.cs
@@ -758,7 +758,7 @@ namespace Ryujinx.UI.Common.Configuration
 
         public void LoadDefault()
         {
-            Logger.EnableFileLog.Value = true;
+            Logger.EnableFileLog.Value = false;
             Graphics.BackendThreading.Value = BackendThreading.Auto;
             Graphics.ResScale.Value = 1;
             Graphics.ResScaleCustom.Value = 1.0f;
@@ -768,9 +768,9 @@ namespace Ryujinx.UI.Common.Configuration
             Graphics.PreferredGpu.Value = "";
             Graphics.ShadersDumpPath.Value = "";
             Logger.EnableDebug.Value = false;
-            Logger.EnableStub.Value = true;
-            Logger.EnableInfo.Value = true;
-            Logger.EnableWarn.Value = true;
+            Logger.EnableStub.Value = false;
+            Logger.EnableInfo.Value = false;
+            Logger.EnableWarn.Value = false;
             Logger.EnableError.Value = true;
             Logger.EnableTrace.Value = false;
             Logger.EnableGuest.Value = true;
@@ -782,8 +782,8 @@ namespace Ryujinx.UI.Common.Configuration
             System.TimeZone.Value = "UTC";
             System.SystemTimeOffset.Value = 0;
             System.EnableDockedMode.Value = true;
-            EnableDiscordIntegration.Value = true;
-            CheckUpdatesOnStart.Value = true;
+            EnableDiscordIntegration.Value = false;
+            CheckUpdatesOnStart.Value = false;
             ShowConfirmExit.Value = true;
             HideCursor.Value = HideCursorMode.OnIdle;
             Graphics.EnableVsync.Value = true;
diff --git a/src/Ryujinx/Modules/Updater/Updater.cs b/src/Ryujinx/Modules/Updater/Updater.cs
index d8346c8..fccda3b 100644
--- a/src/Ryujinx/Modules/Updater/Updater.cs
+++ b/src/Ryujinx/Modules/Updater/Updater.cs
@@ -637,6 +637,7 @@ namespace Ryujinx.Modules
 
         public static bool CanUpdate(bool showWarnings)
         {
+           return false;
 #if !DISABLE_UPDATER
             if (!NetworkInterface.GetIsNetworkAvailable())
             {
